Its.Log.UnitTests/TelemetryTests.cs

```diff
@@ -107,5 +107,18 @@ public void Telemetry_events_contain_an_operation_name()
                            .Be("hello!");
         }
 
+        [Test]
+        public async Task The_user_identifier_can_be_specified_programmatically()
+        {
+            using (Log.With<Telemetry>(t => t.UserIdentifier = "epic_user").Enter(() => { }))

```

Comment:
Current thinking was that in this cause it would just not be set. As it is up to the consumer of the package to set then I'm not sure there is a meaningful default that can be provided here.